pkg/config/seelog/seelog_config.go

```diff
@@ -56,18 +55,22 @@ func (c *SeelogConfig) Render() (string, error) {
 	return b.String(), err
 }
 
+func getJSONSyslogFormat(name string) string {
```

Comment:
let's document this function inline?

In particular, to make it explicit that we _want_ this string to be HTML-escaped by `html/template`, especially given that there are no unit tests on this logic

Reply:
Done
> let's document this function inline?
> 
> In particular, to make it explicit that we _want_ this string to be HTML-escaped by `html/template`, especially given that there are no unit tests on this logic